src/sentry/backup/comparators.py

```diff
@@ -785,6 +785,9 @@ def get_default_comparators():
             "sentry.orgauthtoken": [
                 HashObfuscatingComparator("token_hashed", "token_last_characters")
             ],
+            "sentry.dashboardwidgetqueryondemand": [
+                DateUpdatedComparator("date_added", "date_modified")
```

Comment:
I believe this should only be `date_modified`, not `date_added`? The purpose of this comparator is to ensure that future updates still pass comparison. But the `date_added` should not change when importing, so no need to add it here.

Reply:
hmm yeah, I believe this was constantly breaking before since `date_added` was `auto_now_add` which works a bit weirdly in django, I'll take it out.